src/Console/Commands/SynchroniseTranslationsCommand.php

```diff
@@ -67,30 +77,70 @@ public function handle()
     {
         $languages = array_keys($this->translation->allLanguages()->toArray());
 
-        $fromDriver = $this->anticipate(__('translation::translation.prompt_from_driver'), $this->drivers);
-        if (! in_array($fromDriver, $this->drivers)) {
-            return $this->error(__('translation::translation.invalid_driver'));
+        // If a valid from driver has been specified as an argument.
+        if ($this->argument('from') && in_array($this->argument('from'), $this->drivers)) {
+            $this->fromDriver = $this->argument('from');
         }
 
-        $toDriver = $this->anticipate(__('translation::translation.prompt_to_driver'), $this->drivers);
-        if (! in_array($toDriver, $this->drivers)) {
-            return $this->error(__('translation::translation.invalid_driver'));
+        // When the from driver will be entered manually or if the argument is invalid.
+        else {
+            $this->fromDriver = $this->anticipate(__('translation::translation.prompt_from_driver'), $this->drivers);
+
+            if (! in_array($this->fromDriver, $this->drivers)) {
+                return $this->error(__('translation::translation.invalid_driver'));
+            }
         }
 
-        $language = $this->anticipate(__('translation::translation.prompt_language_if_any'), $languages);
-        if ($language && ! in_array($language, $languages)) {
-            return $this->error(__('translation::translation.invalid_language'));
+        // Create the driver.
+        $this->fromDriver = $this->createDriver($this->fromDriver);
+
+        // When the to driver has been specified.
+        if ($this->argument('to')) {
```

Comment:
```suggestion
        if ($this->argument('to') && in_array($this->argument('to'), $this->drivers)) {
```
I think we need some extra validation in here?